eng/pipelines/templates/steps/dependency.tests.yml

```diff
@@ -0,0 +1,47 @@
+parameters:
+- name: ProjectFilesOutputFolder
+  type: string
+  default: "$(Build.ArtifactStagingDirectory)"
+- name: NumOfTestsPerJob
```

Comment:
```suggestion
- name: NumOfTestsProjectsPerJob
```

Reply:
I would do either `NumTestProjectsPerJob` or `NumberOfTestProjectsPerJob`

Reply:
Change to `NumOfTestProjectsPerJob`